sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/WrapperSdkExceptionManagerTest.java

```diff
@@ -179,43 +179,51 @@ public void saveWrapperSdkCrash() throws JSONException, IOException {
     }
 
     @Test
+    @PrepareForTest({android.util.Log.class})
     public void saveWrapperSdkCrashWithJavaThrowable() throws JSONException, IOException {
         LogSerializer logSerializer = Mockito.mock(LogSerializer.class);
         when(logSerializer.serializeLog(any(ManagedErrorLog.class))).thenReturn("mock");
         Crashes.getInstance().setLogSerializer(logSerializer);
         byte[] data = new byte[]{'d'};
         Throwable throwable = new Throwable();
+        mockStatic(android.util.Log.class);
+        String stackTrace = "sample stacktrace";
+        Mockito.when(android.util.Log.getStackTraceString(any(Throwable.class))).thenReturn(stackTrace);
         WrapperSdkExceptionManager.saveWrapperException(Thread.currentThread(), throwable, new Exception(), data);
         verifyStatic();
         FileManager.writeObject(any(File.class), eq(data));
         verifyStatic();
-        FileManager.writeObject(any(File.class), eq(throwable));
+        FileManager.write(any(File.class), eq(stackTrace));
```

Comment:
We need to remove readObject and writeObject altogether from FileManager, this contains vulnerable code and needs to be cleaned up (yeah its not the right location to put this review comment but if I do it as a global review comment it won't have a resolve button and might be missed, already happened).